devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -121,7 +125,7 @@ private getCommandClassVersions() {
 def parse(String description) {
 	def results = []
 	if (description.startsWith("Err")) {
-		results << createEvent(descriptionText: description, displayed: true)
+		results += createEvent(descriptionText: description, displayed: true)
```

Comment:
What’s the reasoning behind this change?

Reply:
'<<' created double nested lists, which then it leaded to events not being returned correctly from parse() method (they weren't shown in IDE/both mobile apps)

Reply:
@kianooshST is this something that bothers you, and I should rework it?

Reply:
No that is fine, I was just curious about the change.